libs/clients/statistics/src/lib/statistics.service.ts

```diff
@@ -22,16 +27,33 @@ export class StatisticsClientService {
     private fetch: EnhancedFetchAPI,
     @Inject(LOGGER_PROVIDER)
     private logger: Logger,
+    private ultravioletRadiationService: UltravioletRadiationClientService,
   ) {}
 
   async getMultipleStatistics(query: GetStatisticsQuery) {
     try {
-      const sourceData = await getStatisticsFromSource(
-        this.fetch,
-        this.config?.sourceDataPaths?.split(','),
-      )
+      const promises = [
+        getStatisticsFromCsvUrls(
+          this.fetch,
+          this.config?.sourceDataPaths?.split(','),
+        ),
+      ]
+
+      if (query.sourceDataKeys.includes(LATEST_UV_MEASUREMENT_KEY)) {
+        promises.push(this.ultravioletRadiationService.getLatestMeasurement())
+      }
+      if (query.sourceDataKeys.includes(UV_MEASUREMENT_SERIES_KEY)) {
+        promises.push(this.ultravioletRadiationService.getMeasurementSeries())
```

Comment:
Is there any caching in place for these requests?

Reply:
No, my idea was to try and get this in as is and then implement caching in a follow up PR